test/data/genesis_delegates.json

```diff
@@ -17,7 +17,7 @@
 				"9d3058175acab969f41ad9b86f7a2926c74258670fe56b37c429c01fca9f2f0f",
 			"username": "genesis_1",
 			"password": "elephant tree paris dragon chair galaxy",
```

Comment:
Not directly a part of this issue, but we should be able to use a default password approach here too to avoid the redundancy of storing the default password a hundred times.